sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_content_events_v1/query.sql

```diff
@@ -13,16 +13,8 @@ SELECT
     ) AS jsonPayload
   )
 FROM
-  `moz-fx-fxa-prod-0712.fxa_prod_logs.docker_fxa_content_20*`
+  `moz-fx-fxa-prod-0712.fxa_prod_logs.docker_fxa_content`
```

Comment:
Wasn't date-sharded table used here to avoid missing data resulting from schema incompatibilities detected in BQ Cloud Logging sink? I might be completely wrong but IIRC I heard @jbuck talking about this at some point.

Reply:
In the past there was an issue when we tried to switch over to the partitioned table. This is why for this specific table we ended up reverting back to using the sharded table. There's still very small inconsistencies between the two tables. However, the error tables for both look almost identical (in terms of schema error counts) as per my inside the ticket (DENG-537).